src/Tmds.Ssh/SftpChannel.cs

```diff
@@ -949,7 +949,7 @@ public async ValueTask DownloadDirectoryEntriesAsync(string remoteDirPath, strin
                         break;
                     case UnixFileType.RegularFile:
                         lastDirectory = EnsureParentDirectory(lastDirectory, item.LocalPath);
-                        onGoing.Enqueue(DownloadFileAsync(item.RemotePath, item.LocalPath, item.Length, overwrite, item.Permissions, throwIfNotFound: false, cancellationToken));
+                        onGoing.Enqueue(DownloadFileAsync(item.RemotePath, item.LocalPath, null, item.Length, overwrite, item.Permissions, throwIfNotFound: false, cancellationToken));
```

Comment:
Please include the arg name: `destination: null`.

Reply:
done